muranotest/Dockerfile

```diff
@@ -13,9 +16,10 @@ RUN python setup.py install
 RUN pip install nose tempest-lib
 WORKDIR /opt/murano
 RUN pip install -r test-requirements.txt
+COPY requirements.txt /opt/murano/requirements2.txt
```

Comment:
wht do you need to make a copy of the requirements file? The same for config.conf.